src/app/features/balances-list/components/collectible-assets.tsx

```diff
@@ -18,15 +18,16 @@ export const CollectibleAssets = memo((props: StackProps) => {
       {keys.map(key => {
```

Comment:
Maybe it'd be better to use a `.filter()` to exclude 0 amounts, than doing this logic in the JSX?

Reply:
Ok, yep, I can do it that way. I'll change it.